api/services/mapping/utils.js

```diff
@@ -6,20 +6,26 @@ module.exports = {
    * @param {string} key
    * @param {Array} data array of data to be converted
    * @param {(item) => Array} fn function to apply to each data item
+   * @param filterDeleted If true then all items with the isDeleted property set
+   *                      to true will be filtered from the array before applying
+   *                      the given function to apply.
    * @returns {Array | Object}
    */
-  toList: (key, data, fn) => {
+  toList: (key, data, fn, filterDeleted = false) => {
```

Comment:
May be use a named argument for better clarity in the code when the function is called, what do you think ?

Reply:
Are you talking about this @vmarseguerra ?
![image](https://user-images.githubusercontent.com/985903/210522125-29d2e418-0cce-4da0-b22a-f7df5a03a322.png)

Reply:
Yes like that